src/hooks/useMarkdownStyle.ts

```diff
@@ -0,0 +1,47 @@
+import type {MarkdownStyle} from '@expensify/react-native-live-markdown';
+import {useMemo} from 'react';
+import variables from '@styles/variables';
+import useTheme from './useTheme';
+
+function useMarkdownStyle(): MarkdownStyle {
+    const theme = useTheme();
+
+    const markdownStyle = useMemo(
+        () => ({
+            syntax: {
+                color: theme.syntax,
+            },
+            link: {
+                color: theme.link,
+            },
+            h1: {
+                fontSize: variables.fontSizeh1,
+            },
+            quote: {
+                borderColor: theme.border,
+                borderWidth: 4,
+                marginLeft: 0,
+                paddingLeft: 6,
+            },
+            code: {
+                color: theme.text,
+                backgroundColor: theme.textBackground,
```

Comment:
We might consider using a different background color for code blocks so that the markdown in composer appears more consistent with the markdown in sent messages.